src/torch_tomogram/tomogram.py

```diff
@@ -19,6 +25,7 @@ def __init__(
         tilt_axis_angle: torch.Tensor,
         sample_translations: torch.Tensor,
         images: torch.Tensor,  # (b, h, w)
+        pixel_spacing: float | None = None,
```

Comment:
```suggestion
        pixel_spacing: float,
```

great that you added this! I don't think this should be optional as once CTF is implemented it will be necessary to get this right for proper defocus determination